nuxeo-user-group-management/nuxeo-edit-password.html

```diff
@@ -31,8 +31,8 @@
     <nuxeo-input id="password" type="password" label="[[i18n('editPassword.password')]]" value="{{password}}"
                  required></nuxeo-input>
 
-    <nuxeo-input id="passwordConfirmation" type="password" label="[[i18n('editPassword.verify')]]"
-                 required auto-validate pattern="[[password]]" error-message="[[i18n('editPassword.noMatch')]]"></nuxeo-input>
+    <nuxeo-input id="passwordConfirmation" type="password" label="[[i18n('editPassword.verify')]]" value="{{_confirmationPassword}}"
+                 required auto-validate pattern="[[escapeRegExp(password)]]" error-message="[[_computeErrorMessage(password, _confirmationPassword)]]"></nuxeo-input>
```

Comment:
and just `required auto-validate pattern="[[escapeRegExp(password)]]" error-message="[[i18n('editPassword.noMatch')]]"`

`_computeErrorMessage` calls `_getValidity` each time value changes and `auto-validate pattern="[[escapeRegExp(password)]]"` also validates, this is kind of repetitive

Reply:
Thanks for your input! 👍 
Despite not using the approach suggested, in order to avoid both input elements having error messages, I simplified the logic because, in fact, there is no need to call validation method to choose the error message.